### DISCUSSION TITLE
Failure to build the PX4 jsbsim_bridge

### DISCUSSION THREAD

**Discussion by oricabel** [JSBSim-Team/jsbsim#653] [Q&A] [answered]
Hello!
I am currently trying to run the jsbsim simulation using the PX4 jsbsim bridge but during the bulid, one package fails.

input: "
$ catkin build jsbsim_bridge
"

output: "
Failed     << jsbsim_bridge:make           [ Exited with code 2 ]                                                          
Failed    <<< jsbsim_bridge                [ 1.4 seconds ]                                                                 
[build] Summary: 4 of 5 packages succeeded.                                                                                
[build]   Ignored:   2 packages were skipped or are skiplisted.                                                            
[build]   Warnings:  None.                                                                                                 
[build]   Abandoned: None.                                                                                                 
[build]   Failed:    1 packages failed.                                                                                    
[build] Runtime: 3.9 seconds total.                "

Please, can anyone shed some light as to why the build fails?
I am using ubuntu 18.04, I have followed "ROS with MAVROS installation guide" and used ROS melodic.

Thanks!

**Comment by Jaeyoung-Lim** [comment, marked as answer]
@oricabel Are you trying to build jsbsim bridge? Or this repository?

The px4 jsbsim bridge is not part of this repository

**Reply by oricabel** [reply]
Hi again! Yes i am trying to build the jsbsim_bridge, and I'm kinda new in here. I haven't seen a discussion tab in the jsbsim_bridge repository so I thought to try my luck here.

**Reply by Jaeyoung-Lim** [reply]
@oricabel please post a issue in the jsbsim bridge repository and share the full log

**Reply by oricabel** [reply]
Ok, thanks!
